apps/snitch_core/config/elasticsearch.exs

```diff
@@ -0,0 +1,56 @@
+use Mix.Config
+
+config :snitch_core, Snitch.Tools.ElasticsearchCluster,
+  # The URL where Elasticsearch is hosted on your system
+  url: "http://localhost:9200",
```

Comment:
We should make this URL configurable (Maybe in environment variables.). If the size of the index grows later on. We would eventually want to move the ES indexes in a separate AWS instance.

Reply:
@vikramjadon921995 I will update that as per elasticsearch prod docker configuration.